handlers/deployment.rb

```diff
@@ -34,9 +34,11 @@ class Deployment < Handler
       #        and in use for all K8s deployed services
       route(/^(deploy)\s*(.*)/, :tag_deploy, command: true, help: {"deploy REPO" => "Updates the production-release tag on zooniverse/REPO"})
       route(/^(migrate)\s*(.*)/, :tag_migrate, command: true, help: {"migrate REPO" => "Updates the production-migrate tag on zooniverse/REPO"})
-      route(/^(status\s*all)/, :status_all, command: true, help: {'staus all' => 'Returns the deployment status for all previously deployed $REPO_NAMES.'})
+      route(/^(status\s*all)/, :status_all, command: true, help: {'status all' => 'Returns the deployment status for all previously deployed $REPO_NAMES.'})
       route(/^(status|version)\s+(?!all)(.+)/, :status, command: true, help: {'status REPO_NAME' => 'Returns the state of commits not deployed for the $REPO_NAME.'})
       route(/^(history)\s(.+)/, :commit_history, command: true, help: {'history REPO_NAME' => 'Returns the last deployed commit history (max 10) .'})
+      route(/^(static)\s(.+)/, :static, command: true, help: {'static ACTION' => 'Runs the specified static repo action'})
+      route(/^(action)\s(.+)/, :action, command: true, help: {'action REPO FILE BRANCH' => 'Runs the specified repo action on the specified branch'})
```

Comment:
Is this really type of generic action trigger event needed? 

I'm wondering what the direct use case is here, rather than say general need to run gh workflows via chat ops. 

In the past we've used lita to define specific chat ops triggers and targets which are quite targetted, e.g. `lita deploy caesar` etc vs provide the abililty to run all the actions. 

It could open up all the GH actions we have setup could be run arbitrarily via these lita cmds, e.g. production db migration or rake task etc if they have a corresponding gh action. 

Do we want more control over these types of trigger events on GH action workflows? 

Do we want some sort of control over which actions can be run?

Would this be better served by using the GH UI to trigger workflows via the `workflow_dispatch` trigger for event tracking / auditing and a nice UI with authorization for doing so?

E.g. https://github.com/zooniverse/panoptes/blob/98214ed0425c1e5e66d26c5bfaa90eef6d580cd6/.github/workflows/deploy_production.yml#L7
 
<img width="989" alt="Screenshot 2022-05-12 at 13 31 52" src="https://user-images.githubusercontent.com/295329/168075338-ed7f65de-0978-489e-82e4-5783cdd46306.png">

Reply:
I feel like there's really only one thing that can happen to them. That said, if I'm moving the nginx deploy to the standard `production-release` tag, then the `static` namespace becomes less relevant. 

With regard to authentication, adding some kind of RBAC interface between Lita's input and output would be nice, but probably not worth the effort. Access to Slack and specifically certain channels should probably be more restricted first if that's something we're worried about. 

Fundamentally, this PR was intended to add convenience features only, and now is the time to decide how we want our future interactions with CI/CD to look. If we're happier with using the GH UI to run these jobs instead of Lita for whatever reasons, cool. I'm not too opinionated in any direction: while I do like the chatops we've got, it might be stretching Lita a bit thin to start shoving more in.

I'll rename the ingress application route and ditch the generic route and we can think about the above.